components/public-api/go/client/client.go

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package client
+
+import (
+	"errors"
+	"fmt"
+	"github.com/bufbuild/connect-go"
+	gitpod_experimental_v1connect "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v1/v1connect"
+	"net/http"
+)
+
+type Gitpod struct {
+	cfg *options
+
+	Workspaces           gitpod_experimental_v1connect.WorkspacesServiceClient
```

Comment:
Not sure if this discussion is in-scope for this PR, but: I noticed all of those are experimental APIs.
I think it can make sense to share the same client with experimental and stable APIs, but I wonder if we should mark them as such? E.g. offer them under `client.Experimental.Workspaces`, while also having a `client.Workspaces`? :thinking:

Reply:
Good point. I think that change makes sense and I'll make it in a subsequent PR